pages.fr/linux/userdel.md

```diff
@@ -1,8 +1,17 @@
 # userdel
 
-> Efface un utilisateur.
+> Supprime un compte utilisateur.
```

Comment:
Shouldn't this `remove a user from a group` be mentioned too here?

Reply:
Perhaps I missed something in the [man page for userdel](https://linux.die.net/man/8/userdel), but I don't think `userdel` allows you to remove a user from a group (although a side-effect of deleting a user is that the user is removed from all groups it belonged to). That would be `usermod`, afaik. There is a `userdel` option to delete a group with the same name as the deleted user though, but that doesn't equate to "remove a user from a group".

Should I translate this part of the sentence to match the English page or should we edit the English version?

Reply:
> Perhaps I missed something in the [man page for userdel](https://linux.die.net/man/8/userdel), but I don't think `userdel` allows you to remove a user from a group (although a side-effect of deleting a user is that the user is removed from all groups it belonged to). That would be `usermod`, afaik. There is a `userdel` option to delete a group with the same name as the deleted user though, but that doesn't equate to "remove a user from a group".
> 
> Should I translate this part of the sentence to match the English page or should we edit the English version?

I guess we can update the sentence to match the English page.

Reply:
> Should I translate this part of the sentence to match the English page or should we edit the English version?

@mcombeau, there are no formal rules about it right now: you can modify original pages to sync with translations and vice versa.

Reply:
I have translated the sentence to match the English page until a verdict is reached about `userdel`. Thank you @kbdharun and @EmilySeville7cfg for your clarifications !